From e444f500ff40ce241fe1de72e2ab3b3799099d62 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Thu, 7 Dec 2017 16:32:17 -0800
Subject: [PATCH] Move socket to /run/postgresql$VER

By default, the UNIX domain socket for postgresql is created in /tmp.
This fits more appropriately under /run/postgresql$VER.

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 src/include/pg_config_manual.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/pg_config_manual.h b/src/include/pg_config_manual.h
index 743401c..1fd7716 100644
--- a/src/include/pg_config_manual.h
+++ b/src/include/pg_config_manual.h
@@ -179,7 +179,7 @@
  * here's where to twiddle it.  You can also override this at runtime
  * with the postmaster's -k switch.
  */
-#define DEFAULT_PGSOCKET_DIR  "/tmp"
+#define DEFAULT_PGSOCKET_DIR  "/run/postgresql12"
 
 /*
  * This is the default event source for Windows event log.
-- 
2.25.0

